ferrocene/ignored-tests.toml

```diff
@@ -49,6 +49,11 @@ tests = [
 targets = ["aarch64-unknown-ferrocenecoretest"]
 reason = "QEMU user space emulation outputs an extra message when an abort happens"
 
+[["tests/ui"]]
+tests = ["tests/mir-opt/inline/polymorphic_recursion.rs"]
+targets = ["aarch64-unknown-ferrocenecoretest", "x86_64-unknown-linux-gnu", "aarch64-unknown-none"]
+reason = "The test is extremely slow due to debug assertions being enabled"
```

Comment:
Can we do this as a `// ignore-test The test is extremely slow due to debug assertions being enabled` in the file itself?

Reply:
sure